core/src/main/java/com/google/cloud/sql/core/CloudSqlInstance.java

```diff
@@ -224,65 +222,55 @@ void forceRefresh() {
    * value of currentInstanceData and schedules the next refresh shortly before the information
    * would expire.
    */
-  private InstanceData performRefresh() throws InterruptedException, ExecutionException {
+  @VisibleForTesting
+  InstanceData performRefresh() throws InterruptedException, ExecutionException {
```

Comment:
Why are we making this public? It might be a little more work, but we should try to test this through the public interface.

Reply:
Fixed. These are private again.